client/src/main/java/com/paypal/selion/annotations/MobileTest.java

```diff
@@ -124,5 +124,11 @@
      * This parameter specifies to execute mobile test cases using respective mobile driver.
      */
     String mobileNodeType() default "";
+    
+    /**
+     * This parameter specifies the name of the browser
+     * 
+     */
 
+    String browserName() default "";
```

Comment:
I'm not sure if this is ideal to have an additional field for browserName in the mobile use case.  The appName field is intended to work even for browsers values (e.g.  "safari").

Reply:
As far as I know appium would expect a separate capability "browserName" in order to create a browser based session and "app" to create an app based session. 

If you try to set app=safari, appium will complain that "app name does not have extension .app" or in case of chrome ".apk", that's why we need this capability to set the browserName separately.

Other observation- testing infrastructure providers such as sauce they also expect a browserName capability in order to create a browser based session - https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/

Reply:
What is even more annoying is in the case of an app or hybrid test, they still expect a capability `browserName: ""`, I believe.